lib/Bindings/Python/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 
 include(AddMLIRPython)
 add_custom_target(CIRCTBindingsPython)
+add_dependencies(CIRCTBindingsPython MLIRBindingsPythonExtension)
+add_dependencies(CIRCTBindingsPython MLIRBindingsPythonSources)
```

Comment:
These aren't necessary, but I think this is the right way to get CMake to rebuild the upstream Python sources and library when they change.